### PR TITLE
Create invitation-for-cfp-template.md

### PR DESCRIPTION
**Pull request by vipulgupta2048**
@anistark @palnabarun please review
Content inspired by: @chocos10

### CODE REVIEW

**Comment by anistark** [comment]
Looks good to me. Let's merge this.

File: templates/emails/invitation-for-cfp-template.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Template for the invitation for CFP
+
+Hello {{POTENIAL_SPEAKER_FIRST_NAME}},
+
+We hope you are in good health and doing well. This email is regarding PyCon India 2020 inviting you to submit a proposal for our open call for proposals. PyCon India is the premier conference for the Python programing language in India and has been running for well over 12 years, scaling in both size and folks attending. Last year, we had over 300+ proposals! What's better, we are completely non-profit and volunteer-run through and through. Helping promote Python through many communities around India.
```
Comment (palnabarun) [inline templates/emails/invitation-for-cfp-template.md:5]:
```suggestion
We hope you are in good health and doing well. This email is regarding PyCon India 2020 inviting you to submit a proposal for our open call for proposals. PyCon India is the premier conference for the Python programing language in India and has been running for well over 12 years, scaling in both size and attendees. Last year, we had over 300+ proposals! What's better, we are completely non-profit and volunteer-run through and through, helping promote Python through many communities around India. 
```

File: templates/emails/invitation-for-cfp-template.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+## Template for the invitation for CFP
+
+Hello {{POTENIAL_SPEAKER_FIRST_NAME}},
+
+We hope you are in good health and doing well. This email is regarding PyCon India 2020 inviting you to submit a proposal for our open call for proposals. PyCon India is the premier conference for the Python programing language in India and has been running for well over 12 years, scaling in both size and folks attending. Last year, we had over 300+ proposals! What's better, we are completely non-profit and volunteer-run through and through. Helping promote Python through many communities around India. 
+
+This year with the current situation [1] We, the conference organizers, are taking this as a unique opportunity to be able to put more efforts into making this year's conference bigger, more inclusive, and much more accessible to our lovely attendees from all over the world in ways more than one. And, we need your expertise in that. We would be glad to see your proposal at our conference. We believe your experience and insights with Python and/or with Python community will be a critical part of making this conference a huge success.
```
Comment (palnabarun) [inline templates/emails/invitation-for-cfp-template.md:7]:
```suggestion
This year, with the current situation [1], we, the conference organizers, are taking this as a unique opportunity to be able to put more efforts into making this year's conference bigger, more inclusive, and much more accessible to our lovely attendees from all over the world in ways more than one. And, we need your expertise in that. We would be glad to see your proposal at our conference. We believe your experience and insights with Python and with Python community will be a critical part of making this conference a huge success.
```

Something doesn't sound right with the first sentence. It is too long and has too much content. Can we split it?

File: templates/emails/invitation-for-cfp-template.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+We really hope to see your proposals. The deadline to submit is on 14th August 2020 and we release the schedule by 1st September. Please do reach out to us if you have any questions. If you like to learn more about writing proposals for talks and workshops, you can read one of the blogs below.
```
Comment (palnabarun) [inline templates/emails/invitation-for-cfp-template.md:9]:
```suggestion
We really hope to see your proposals. The deadline to submit is 14th August 2020 and the schedule would be released on 1st September. Please do reach out to us if you have any questions. If you like to learn more about writing proposals for talks and workshops, you can read one of the blogs below. 
```

File: templates/emails/invitation-for-cfp-template.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+We really hope to see your proposals. The deadline to submit is on 14th August 2020 and we release the schedule by 1st September. Please do reach out to us if you have any questions. If you like to learn more about writing proposals for talks and workshops, you can read one of the blogs below. 
+
+- [https://in.pycon.org/blog/2020/2020-call-for-proposals-announcement.html](https://in.pycon.org/blog/2020/2020-call-for-proposals-announcement.html)
+- [https://in.pycon.org/blog/2020/2020-workshop-cfp-announcement.html](https://in.pycon.org/blog/2020/2020-workshop-cfp-announcement.html)
+
+Thank your for taking the time to read through and hope to see you as a speaker at PyCon India 2020
```
Comment (palnabarun) [inline templates/emails/invitation-for-cfp-template.md:14]:
```suggestion
Thank your for taking the time to read through and hope to see you as a speaker at PyCon India 2020.
```

**Comment by vipulgupta2048** [comment]
Thanks @palnabarun, didn't grammarly it. Will do.

**Comment by anistark** [comment]
@vipulgupta2048 Can we wrap this up?

**Comment by vipulgupta2048** [comment]
@anistark Wrapped up, LGTM from my side. Squash and merge. Thanks!

**Comment by anistark** [comment]
@palnabarun Please merge if it looks okay to you.

File: templates/emails/invitation-for-cfp-template.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+## Template for the invitation for CFP
+
+Hello {{POTENIAL_SPEAKER_FIRST_NAME}},
+
+We hope you are in good health and doing well. This email is regarding PyCon India 2020 inviting you to submit a proposal for our open call for proposals. PyCon India is the premier conference for the Python programing language in India and has been running for well over 12 years, scaling in both size and attendees. Last year, we had over 300+ proposals! What's better, we are completely non-profit and volunteer-run through and through, helping promote Python through many communities around India. 
+
+With the current situations, we, the conference organizers, are taking the unique opportunity to put more efforts into making our conference bigger, more inclusive, and much more accessible to all our attendees. Working on this new direction, we would be glad to see your proposal at our conference. We believe your experience, expertise and insights with Python and/or with Python community will be a critical part of making this conference a huge success.
```
Comment (palnabarun) [inline templates/emails/invitation-for-cfp-template.md:7]:
```suggestion
With the current situation, we, the conference organizers, are taking the unique opportunity to put more effort into making our conference bigger, more inclusive, and much more accessible to all our attendees. Working on this new direction, we would be glad to see your proposal at our conference. We believe your experience, expertise, and insights with Python and/or with Python community will be a critical part of making this conference a huge success. 
```

File: templates/emails/invitation-for-cfp-template.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+With the current situations, we, the conference organizers, are taking the unique opportunity to put more efforts into making our conference bigger, more inclusive, and much more accessible to all our attendees. Working on this new direction, we would be glad to see your proposal at our conference. We believe your experience, expertise and insights with Python and/or with Python community will be a critical part of making this conference a huge success. 
+
+We really hope to see your proposals. The deadline to submit is 14th August 2020 and the schedule would be released on 1st September. Please do reach out to us if you have any questions. If you like to learn more about writing proposals for talks and workshops, you can read one of the blogs below.
```
Comment (palnabarun) [inline templates/emails/invitation-for-cfp-template.md:9]:
```suggestion
We really hope to see your proposal(s). The deadline to submit is 14th August 2020 and the schedule would be released on 1st September. Please do reach out to us if you have any questions. If you like to learn more about writing proposals for talks and workshops, you can read one of the blogs below:
```

Comment (vipulgupta2048) [reply templates/emails/invitation-for-cfp-template.md:9]: Good one.

**Comment by vipulgupta2048** [comment]
@palnabarun Ready for launch :rocket:

**Comment by anistark** [comment]
@palnabarun Is this good to be merged now? We need to send these mails as soon as possible.

**Comment by palnabarun** [comment]
Thank you @vipulgupta2048.

Can someone having access to this repository `squash and merge` this PR?

**Pull request merged**